daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -577,10 +577,11 @@ enum BuiltinFunction {
   SCALE_BIGNUMERIC = 137; // *Available in versions >= 1.13*
   PRECISION_BIGNUMERIC = 138; // *Available in versions >= 1.13*
   ADD_BIGNUMERIC = 139; // *Available in versions >= 1.13*
+  ADD_BIGNUMERIC = 139; // *Available in versions >= 1.13*
```

Comment:
I think this may be duplicated by accident.

Reply:
```suggestion
  ADD_BIGNUMERIC = 139; // *Available in versions >= 1.13*
```

Reply:
good catch.